### DISCUSSION TITLE
yup-fast-check?

### DISCUSSION THREAD

**Discussion by peniqliotuv** [dubzzz/fast-check#5195] [Ideas]
My company uses `yup` heavily and I recently introduced property testing there. I'm finding myself frustrated at the lack of an equivalent `yup-fast-check` library. I'm at the point where I'm going to build it myself, but I wanted to gauge the interest of the community and maintainers of `fast-check` to see if there have been discussions around this in the past.

**Comment by dubzzz** [comment]
At the moment, there is no plan to have a yup-fast-check in fast-check repository itself. That said happy to welcome a new community package! We do already have some community packages for zod, fp-ts... (more in [Ecosystem](https://fast-check.dev/docs/ecosystem/)).

If you start working on it and publish a version, please let me know so that I can add it on the page above if you want.

**Reply by peniqliotuv** [reply]
Cool, will keep you in the loop!
